src/classes/GiftSelector.cls

```diff
@@ -43,4 +43,13 @@ public with sharing class GiftSelector {
                 AND Payment_Status__c=: ElevateTransactionStatus.AUTHORIZED.name()
         ];
     }
+
+    public DataImport__c getGiftById(Id giftId) {
```

Comment:
If this is called "getGiftById" should it return an instance of gift instead of a DataImport__c? Or does this class actually maintain that boundary so it's appropriate?

Reply:
That's a good question that I feel like the other developers would be able to better answer. I know for me, I using the naming already established where `getAuthorizedGiftsFor()` on the class returns a `List<DataImport__c>`. Please let me know if you think any adjustments to the naming is necessary here.